spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/ClientRecoveryCompatibilityTests.java

```diff
@@ -44,6 +49,10 @@
 	public void testDefeatRecovery() throws Exception {
 		final Channel channel1 = mock(Channel.class);
 		when(channel1.isOpen()).thenReturn(true);
+		willAnswer(i -> {
+			return null;
```

Comment:
This is left behind from debugging - I needed to figure out where the close() was called from; can be removed.